electron_app/yarn-error.log

```diff
@@ -0,0 +1,905 @@
+Arguments: 
```

Comment:
This file should not be staged

Reply:
good point, I'll .gitignore it

Reply:
fixed

Reply:
It shouldn't really be ignored here either: it's nothing to do with the functionality this PR is adding